js/wallet/walletView.js

```diff
@@ -0,0 +1,17 @@
+class WalletView {
+  constructor(model, target) {
+    this.target = target;
+    this.model = model;
+    this.model.subscribe(this.render.bind(this));
+  }
+
+  render(data) {
+    this.target.walletTotalView.innerHTML = `${data.walletTotal + " 원"}`;
```

Comment:
여기도 템플릿 리터럴을 아래처럼 사용하는게 어떨까요?
```suggestion
    this.target.walletTotalView.innerHTML = `${data.VMCash} 원`;
```